schemas/webpack.d.ts

```diff
@@ -0,0 +1,364 @@
+// Type definitions for Webpack
+// Project: https://webpack.github.io/
+// Definitions by: Nejc Zdovc <https://github.com/NejcZdovc>
+// Definitions: https://github.com/webpack/webpack
+
+
+declare namespace WebpackType {
+    interface Node {
+        Buffer?: boolean;
+        __dirname?: boolean | "mock";
+        __filename?: boolean | "mock";
+        console?: boolean;
+        global?: boolean;
+        process?: boolean
```

Comment:
In [scheme.json](https://github.com/webpack/webpack/blob/master/schemas/webpackOptionsSchema.json#L673) which is a base line for this PR it's says only boolean.